src/docsfy/api/projects.py

```
@@ -0,0 +1,1627 @@
+                )
+                return
+
+        try:
+            await delete_project(
+                project_name,
+                ai_provider=source_provider,
+                ai_model=source_model,
+                owner=owner,
+                branch=branch,
+            )
+        except Exception as exc:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)